http/1.7/modules/ROOT/pages/http-authentication.adoc

```diff
@@ -699,6 +700,227 @@ In the *Configuration XML* editor, the `tls:context`, `tls:trust-store` and `tls
     </tls:context>
 ----
 
+[[oauth2-grant-authentication-connected-apps]]
+== Configure OAuth2 Authorization Code Grant Type Authentication Using Connected Apps
+:page-aliases: connectors::http/http-configure-authorization-code-for-requester.adoc
+
+You can also configure the OAuth2 authorization code grant type authentication using the xref:access-management::connected-apps-developers.adoc[Connected Apps] feature.  
```

Comment:
The "also" assumes that people are reading sequentially. I think users will pick either the previous or this method.